R/job_operations.R

```diff
@@ -117,3 +117,23 @@ updateJob <- function(jobId, ...){
 
   callBatchService(request, batchCredentials, body)
 }
+
+#' Gets job task counts by job state.
+#'
+#' @param jobId The id of the job.
+#'
+#' @return A response containing the task counts of different states.
+#' @examples
```

Comment:
This will need a /dontrun { getJobTasksCount() } or else Travis CI will try to run this function